src/http-gateways/trustless-gateway.md

```diff
@@ -214,7 +214,7 @@ The Body hash MUST match the Multihash from the requested CID.
 
 A CAR stream for the requested
 [application/vnd.ipld.car](https://www.iana.org/assignments/media-types/application/vnd.ipld.car)
-content type (with optional `order` and `dups` params), path and optional
+content type (with optional `order`, `dups` and `skip-leaves` params), path and optional
```

Comment:
:warning: nit: this does not belong to content type. It belongs to URL params.

Rationale:
-  `order` and `dups` are parameters of CAR response format. they do not control the scope of user data that is beign returned, only how the user data is represented in CAR format
- `skip-leaves` proposed here CHANGES the scope of user data returned in response (some blocks won't be returned, not even once), which makes it the same abstraction as `dag-scope` and `entity-bytes`

Reply:
Moved in 131b29d2c39c0f210d1f4449d9d416793350538c